### PR TITLE
style: update text for SLL Tooltip Revised

### PR DESCRIPTION
**Pull request by gabester78**
### SUMMARY

Changed the SSL tooltip text to updated based on the database the user selects.
https://trello.com/c/B4i4wtpq/29-update-ssl-tooltip-text

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

Screen Shot 2021-09-22 at 12 49 32 PM

### TESTING INSTRUCTIONS

- From Welcome screen navigate to Data > Databases
- Click on "+ Database" button
- Click on PostgreSQL or MySql icons or select Amazon Redshift from the supported databases dropdown menu
- Hover over SLL Tooltip at the bottom of the form 

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [x] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -30,7 +30,7 @@ combine_as_imports = true
 include_trailing_comma = true
 line_length = 88
 known_first_party = superset
-known_third_party =alembic,apispec,backoff,bleach,cachelib,celery,click,colorama,cron_descriptor,croniter,cryptography,dateutil,deprecation,flask,flask_appbuilder,flask_babel,flask_caching,flask_compress,flask_jwt_extended,flask_login,flask_migrate,flask_sqlalchemy,flask_talisman,flask_testing,flask_wtf,freezegun,geohash,geopy,graphlib,holidays,humanize,isodate,jinja2,jwt,markdown,markupsafe,marshmallow,marshmallow_enum,msgpack,numpy,pandas,parameterized,parsedatetime,pgsanity,pkg_resources,polyline,prison,progress,pyarrow,pyhive,pyparsing,pytest,pytest_mock,pytz,redis,requests,selenium,setuptools,simplejson,slack,sqlalchemy,sqlalchemy_utils,sqlparse,typing_extensions,urllib3,werkzeug,wtforms,wtforms_json,yaml
```
Comment (hughhhh) [inline setup.cfg:33]: did you want to add `graphlib` here?

File: superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx [hunk 2]
```diff
@@ -480,30 +480,40 @@ const forceSSLField = ({
...
+              name: 'encryption',
+              checked: true,
+              value: changed,
+            },
+          });
+        }}
+      />
+      <span css={toggleStyle}>SSL</span>
+      <InfoTooltip
+        tooltip={t(tooltipText[db?.database_name!])}
```
Comment (AAfghahi) [inline superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx:510]: Hey! using the ! is not an encouraged action. Was this to solve a TypeScript error? Also, what happens if a database name isn't in toolTip text? Might be worth to have a || here or a default option. Let me know if you want help with either of those options.

File: superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx [hunk 3]
```diff
@@ -480,30 +480,40 @@ const forceSSLField = ({
...
-      tooltip={t('SSL Mode "require" will be used.')}
-      placement="right"
-      viewBox="0 -5 24 24"
-    />
-  </div>
-);
+}: FieldPropTypes) => {
+  const tooltipText = {
+    PostgreSQL:
+      'Requires a root certificate authority (public, local, or self-signed). SSL Mode "require" will be used.',
```
Comment (geido) [inline superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx:486]:
```suggestion
      t('Requires a root certificate authority (public, local, or self-signed). SSL Mode "require" will be used.'),
```

File: superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx [hunk 4]
```diff
@@ -480,30 +480,40 @@ const forceSSLField = ({
...
+    MySQL:
+      'Requires a root certificate authority (public, local, or self-signed).',
```
Comment (geido) [inline superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx:488]:
```suggestion
      t('Requires a root certificate authority (public, local, or self-signed).'),
```

File: superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx [hunk 5]
```diff
@@ -480,30 +480,40 @@ const forceSSLField = ({
...
+    'Amazon Redshift':
+      'Requires a root certificate authority (public, local, or self-signed). SSL Mode "verify-ca" will be used.',
```
Comment (geido) [inline superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx:490]:
```suggestion
      t('Requires a root certificate authority (public, local, or self-signed). SSL Mode "verify-ca" will be used.'),
```

**Review by geido — COMMENTED** [review]
Thanks for your contribution! I added some suggestions to include localized strings

File: setup.cfg (hunk 1 above)
Comment (gabester78) [reply setup.cfg:33]: Yes, I was trying to. I couldn't find the right way to install it.

File: superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx (hunk 2 above)
Comment (gabester78) [reply superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx:510]: Yes, it was to solve an error. The code was working on localhost, but the error below wouldn't go away. The data isn't passed through to the tooltip until the user clicks on the card. Typescript assumed there was no data. 
Screen Shot 2021-10-01 at 3 28 58 PM

**Comment by gabester78** [comment]
Closing PR. The feature is being changed back to the same string for all databases the user may choose. [Trello Task](https://trello.com/c/B4i4wtpq/29-update-ssl-tooltip-text)

**Pull request closed without merging**